src/VisualStudio/Core/Def/Implementation/FindReferences/AbstractAdditionalPropertyDefinition.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using System.Linq;
+using Microsoft.VisualStudio.Shell.TableControl;
+
+namespace Microsoft.VisualStudio.LanguageServices.AdditionalProperty
+{
+    /// <summary>
+    /// Column definition for additional properties to be displayed in the Find All References window.
+    /// </summary>
+    internal abstract class AbstractAdditionalPropertyDefinition : TableColumnDefinitionBase
```

Comment:
I think this type specifically deals with FAR columns, it derives from `TableColumnDefinitionBase`. We should change the name to `AbstractCustomColumnDefinition`

```suggestion
    internal abstract class AbstractCustomColumnDefinition : TableColumnDefinitionBase
```